dist/models/user.d.ts

```diff
@@ -5,17 +5,22 @@ import Location from './subModels/shared/location';
 declare namespace User {
     interface Model {
         _id?: Types.ObjectId;
+        title: string;
+        suffix: string;
+        firstName?: string;
+        lastName?: string;
+        middleName?: string;
         addresses?: Location.Model[];
         admin?: boolean;
         birthday?: Date | null;
         clubID?: Types.ObjectId;
         remoteID?: string;
         email?: string;
-        firstName?: string;
         jobTitle?: string;
+        notes?: string;
```

Comment:
Make this an array. Another thing I forgot to mention is clubtec allows for entries of users that are dated. Black book style of club members....

eg: 
- on: Friday April 11th
- by: Dan Hill
- note: Joe Shmo was belligerent at the range

Reply:
Ok so we should add a notes model.